crypto/evp/pmeth_lib.c

```diff
@@ -294,6 +296,21 @@ int EVP_PKEY_meth_add0(const EVP_PKEY_METHOD *pmeth)
     return 1;
 }
 
+void evp_app_cleanup_int(void)
```

Comment:
Perhaps actually add a call to it in `OPENSSL_cleanup`? ;-)
(you'll have to add a declaration in `crypto/include/internal/evp_int.h` too)

Also, may I suggest a rename to `evp_pkey_cleanup_int`?

Reply:
Seems I've already added this stuff in `evp_cleanup_int` which will be called by `OPENSSL_cleanup` and the header file `evp_int.h` was also updated in the commit.

> Also, may I suggest a rename to evp_pkey_cleanup_int

`pkey` is too wide, I think `evp_pkey_meth_cleanup_int` would be better.

Reply:
Humm, now I see the change that adds it...  dunno why github didn't show me that earlier...